src/Microsoft.AspNet.Routing/RouteValueDictionary.cs

```diff
@@ -31,23 +31,24 @@ public RouteValueDictionary()
         }
 
         /// <summary>
-        /// Creates a <see cref="RouteValueDictionary"/> initialized with the provided input value.
+        /// Creates a <see cref="RouteValueDictionary"/> initialized with the specified <paramref name="values"/>.
         /// </summary>
-        /// <param name="obj">An object to initialize the dictionary. The value can be of type
-        /// <see cref="IDictionary{TKey, TValue}"/> or <see cref="IReadOnlyDictionary{TKey, TValue}"/> or
-        /// any other object.
+        /// <param name="values">An object to initialize the dictionary. The value can be of type
+        /// <see cref="IDictionary{TKey, TValue}"/> or <see cref="IReadOnlyDictionary{TKey, TValue}"/>
+        /// or an object with public properties as key-value pairs.
         /// </param>
         /// <remarks>
-        /// If the value is a dictionary, then its entries are copied. Otherwise the object is interpreted as a set
-        /// of key-value-pairs where the property names are keys, and property values are the values, and copied
-        /// into the dictionary. Only public instance non-index properties are considered.
+        /// If the value is a dictionary or other <see cref="IEnumerable{KeyValuePair{string, object}}"/>,
+        /// then its entries are copied. Otherwise the object is interpreted as a set of key-value pairs where the
```

Comment:
suggest (not demand): everything else is now linked; so `object` -> `<see cref="object"/>` here and twice in the `<param/>` description above.